little-books/src/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.scss';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
Move pkg imports to the top and our custom component imports at the bottom

Reply:
pkg imports has been moved to the top